controllers/wavefrontalert_controller.go

```diff
@@ -149,6 +261,16 @@ func (r *WavefrontAlertReconciler) HandleDelete(ctx context.Context, wfAlert *al
 	//Check if any alerts were created with this config
 	if len(wfAlert.Status.Alerts) > 0 {
 		//Call wavefront api and delete the alerts one by one
+		for _, alert := range wfAlert.Status.Alerts {
+			if alert.ID != "" {
+				if err := r.WavefrontClient.DeleteAlert(ctx, alert.ID); err != nil {
+					log.Error(err, "skipping alert deletion", "alertID", alert.ID)
+					// Just skip it for now
```

Comment:
Do you want to consider update the alert state to be error?

Reply:
We will keep this open- There are scenarios where alert itself is not available in the wavefront and we will see if we get any change in the error response from wavefront alert and also keeping alert in error state is going to retry it multiple times(which may be the good case) but for now we will keep this open and doesn't block the other alerts to be deleted